PCLConfig.cmake.in

```diff
@@ -183,6 +183,10 @@ endmacro(find_qhull)
 
 #remove this as soon as libopenni is shipped with FindOpenni.cmake
 macro(find_openni)
+  if(PCL_FIND_QUIETLY)
+    set(openni_FIND_QUIETLY TRUE)
```

Comment:
It doesn't seem to be lowercase in case of OpenNI2. ([PCLConfig.cmake.in#L223](https://github.com/PointCloudLibrary/pcl/blob/master/PCLConfig.cmake.in#L223))
If there is no reason, Please adjust accordingly.
```patch
-    set(openni_FIND_QUIETLY TRUE)
+    set(OpenNI_FIND_QUIETLY TRUE)
```

Reply:
The only reason to use all lowercase is because it should be the same as the first parameter in the call `find_package_handle_standard_args`.  https://github.com/PointCloudLibrary/pcl/blob/41783c7b5d81b3ddb02f4ec68ae1e04d2faaa637/PCLConfig.cmake.in#L211

I'll change them all.

Reply:
I think it is good. Thanks,